nix/ordered-containers.nix

```diff
@@ -0,0 +1,9 @@
+{ mkDerivation, base, containers, stdenv }:
+mkDerivation {
+  pname = "ordered-containers";
+  version = "0.2.2";
+  sha256 = "c75ac7330e70cd5d6ac0062b68033779cf15cd986d4ca20f838e016d466d22c9";
+  libraryHaskellDepends = [ base containers ];
+  description = "Set- and Map-like types that remember the order elements were inserted";
```

Comment:
That's very interesting! Do these data structures perform well? `dhall` currently uses custom types `Dhall.Map` and `Dhall.Set` for this.